plenum/test/checkpoints/test_checkpoint_bounds_after_catchup.py

```diff
@@ -11,32 +14,28 @@ def test_upper_bound_of_checkpoint_after_catchup_is_divisible_by_chk_freq(
         chkFreqPatched, looper, txnPoolNodeSet,
         sdk_pool_handle, sdk_wallet_steward, sdk_wallet_client, tdir,
         tconf, allPluginsPath):
-    sdk_send_random_and_check(looper, txnPoolNodeSet, sdk_pool_handle,
-                              sdk_wallet_client, 4)
-
-    _, new_node = sdk_add_new_steward_and_node(
-        looper, sdk_pool_handle, sdk_wallet_steward,
-        'EpsilonSteward', 'Epsilon', tdir, tconf,
-        allPluginsPath=allPluginsPath)
-    txnPoolNodeSet.append(new_node)
-    looper.run(checkNodesConnected(txnPoolNodeSet))
-    waitNodeDataEquality(looper, new_node, *txnPoolNodeSet[:-1],
-                         exclude_from_check=['check_last_ordered_3pc_backup'])
+    lagging_node = txnPoolNodeSet[-1]
+    with delay_rules_without_processing(lagging_node.nodeIbStasher, cDelay()):
+        sdk_send_random_and_check(looper, txnPoolNodeSet, sdk_pool_handle,
+                                  sdk_wallet_client, tconf.Max3PCBatchSize * CHK_FREQ * 2 + 1)
+    ensure_all_nodes_have_same_data(looper, txnPoolNodeSet)
+    # waitNodeDataEquality(looper, lagging_node, *txnPoolNodeSet[:-1],
+    #                      exclude_from_check=['check_last_ordered_3pc_backup'])
     # Epsilon did not participate in ordering of the batch with EpsilonSteward
     # NYM transaction and the batch with Epsilon NODE transaction.
     # Epsilon got these transactions via catch-up.
 
     sdk_send_random_and_check(looper, txnPoolNodeSet, sdk_pool_handle,
-                              sdk_wallet_client, 1)
+                              sdk_wallet_client, 4 * tconf.Max3PCBatchSize)
```

Comment:
Why send so much more requests?

Reply:
There're only 4 requests. We need to send them to stabilize checkpoint.